src/index.html

```diff
@@ -1,13 +1,70 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+<html lang="en" class="page">
```

Comment:
There is shoul be only 2 blocks for this task, so you shoul remove block like this.